spirv_glsl.cpp

```diff
@@ -3371,6 +3374,13 @@ string CompilerGLSL::access_chain_internal(uint32_t base, const uint32_t *indice
 				expr += ".";
 				expr += to_member_name(*type, index);
 			}
+
+			if (member_is_packed_type(*type, index))
+			{
+				auto &membertype = get<SPIRType>(type->member_types[index]);
+				expr = unpack_expression_type(expr, membertype);
```

Comment:
This could get problematic when we need to support storing to a packed member, because this expression now has turned into l-value.